src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -275,6 +279,38 @@ void FullyConnected::getSupportedDescriptors() {
 
     inDims = isDynamicNode() ? makeDummyInputDims() : getInputShapeAtPort(DATA_ID).getStaticDims();
     outDims = isDynamicNode() ? makeDummyOutputDims(inDims) : getOutputShapeAtPort(0).getStaticDims();
+#ifdef OV_CPU_WITH_MLAS
+    // MLAS doesn't support post-ops fusing and only supports FP32. INT8 is not enabled yet
+    // Disable MLAS when FC could fuse post-ops
+    useMlas = !useSparseWeights &&
+              (inputDataType == memory::data_type::f32 && weightsDataType == memory::data_type::f32) &&
+              fusedWith.empty();
+    auto wgtDims = getInputShapeAtPort(WEIGHTS_ID).getStaticDims();
+    // MLAS cannot find weight dims > 2, e.g. [1,64,9,9] * [10,64,9,9]
+    if (useMlas && wgtDims.size() > 2) {
+        bool allOnes = true;
+        for (size_t i = 2; i < wgtDims.size(); i++) {
+            allOnes = allOnes && wgtDims[i] == 1;
+        }
+        useMlas = useMlas && allOnes;
+    }
+    if (useMlas && withBiases) {
+        const auto& biasDims = getInputShapeAtPort(BIAS_ID).getStaticDims();
+        bool isByChannel = biasDims.back() == outDims.back();
+        for (size_t i = 0; i < biasDims.size() - 1; i++) {
+            isByChannel = isByChannel && biasDims[i] == 1;
+        }
+        useMlas = useMlas && isByChannel;
+    }
+#endif
+    std::string sgemmType = "MLAS";
+#ifdef CPU_DEBUG_CAPS
+    // Select Sgemm type by ENV MLAS/ONEDNN, MLAS is used by default
+    if (getenv("OV_CPU_GEMM")) {
```

Comment:
I think we will need to refactor this debug capability using generic primitives priority mechanism, but this indees should be done in separate PR. For now I would ask to do the following:
1. Please rename the variable to make it aligned with other debug options: "OV_CPU_FC_EXEC_TYPE"
2. Lets just set useMlas=false in case the variable not eqaul MLAS instead of usage of additional string outside debug define.